### PR TITLE
Create my first Portfolio

### PR DESCRIPTION
**Pull request by Wolfmikerson**
Sectio added to the page:

- Toolbars section

- Headline section

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,92 @@
+.margin-none {
+  margin: 0;
+  padding: 0;
+}
+
+/* Defaults Custom Fonts */
+.default-custom-fonts {
+  font-family: Poppins, Georgia, 'Times New Roman', Times, serif;
```
Comment (Berabjesus) [inline css/style.css:10]: - [ ] As described in the project requirement, You must stick to the design as much as possible including fonts. Currently, you have included the `Poppins` font but it is not properly imported. Please have a look at this [StackOverflow thread](https://stackoverflow.com/questions/14676613/how-to-import-google-web-font-in-css-file)  on how to import google fonts and make the appropriate change. You can also check some other ways on how to include fonts. Make sure you do the same for all other font-related stylings too.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio website</title>
+    <link rel="stylesheet" href="css/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+  <body>
```
Comment (Berabjesus) [inline index.html:9]: The `body` currently has a margin that is causing the background image and the content of the page to not fill the frame of the screen. As you can see below there is some gap between the edge of the screen and the content of the web page. Kindly remove the margin from the body by setting it to zero.
image

**Review by Berabjesus — CHANGES_REQUESTED** [review]
# Status: CHANGES REQUIRED :warning: :construction: 
Hey @Wolfmikerson great start to this project 👏 👏. I like how you used reusable CSS classes. I believe that will serve you very well in the long run. Keep it up!

### However, you do have a few changes to carry out before we proceed so please take a look at the things you have to work on below.

## Required Changes ♻️

- [ ] Please import the fonts specified in the design properly. 

- [ ] Remove margin from the body. (Check inline comment below for more description)

**Please Check the inline comments below and work on the requested changes.**

-------
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
-------
#### **If you have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help** 
#### **HAPPY CODING  :computer**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Wolfmikerson , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request ❌ 

You have submitted a project that has a merge conflict with the main branch, please resolve the conflict and request another review. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+  margin-top: 12px;
+  margin-bottom: 136px;
+  padding: 0;
+}
+
+/* nav bar styling */
+.navbar-contener {
+  width: 100%;
+  padding: 10px 0;
+  margin-top: 44px;
```
Comment (hemantso) [inline css/style.css:39]: - [ ]  Optional
Please try to decrease the `margin-top` in the navbar so that it looks similar to Figma design.

**Review by hemantso — APPROVED** [review]
Hi @Wolfmikerson , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- There is a repetition of content in your `README` file please try to fix it.

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**